benchmarks/Thresholds/5.7/CertificatesBenchmarks.Parse_WebPKI_Roots.p90.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
```

Comment:
I find this format a bit unclear tbh. I expected something along those lines

```
{
  "retain" {
    "count": XXX
  },
  "allocations": {
    "total": XXX,
    "large": XXX,
    "small": XXX,
  }
}
```

@hassila Is this something we could get to work?

Reply:
The format is basically an implementation detail, it's machine generated and machine read?

Reply:
I think that's the difference here for us a bit. We want to understand from the file what the different thresholds are so that if somebody just updates the thresholds we understand if we regressed allocations and by how much.

Reply:
https://github.com/ordo-one/package-benchmark/pull/180/commits/0b2a4d853427458cf60739feb04fee30f337eb3c

Reply:
Should be more readable, it's basically:

```bash
hassila@ice ~/g/package-benchmark (threshold-export)> more Thresholds/P90AbsoluteThresholdsBenchmark.P90Date.p90.json
{
  "syscalls" : 1,
  "mallocCountTotal" : 0
}
hassila@ice ~/g/package-benchmark (threshold-export)> 
```

Reply:
Oh yeah that looks lovely! Thanks for the quick turn around on those. Very much appreciated and I think we are now at stage where we can actually finalise the PR. Right @dnadoba ?